app/models/edm/web_resource.rb

```diff
@@ -140,6 +147,13 @@ def europeana_supported_media_mime_type
       end
     end
 
+    def media_size
+      limit = MAX_MEDIA_SIZE.to_f
+      if media.file.size.to_f > limit
+        errors.add(:media, I18n.t('contribute.form.validation.media_size',size: limit/(1000*1000)))
```

Comment:
Use the `number_to_human_size` helper here.

Reply:
that adds the units in so it comes out as '50 MB' instead of just 50, but yeah I think it's better so I removed the unit from localeapp